### ISSUE TITLE
p2a

### ISSUE THREAD

**Issue by tremendapithon** [IIC1253/IIC1253-2024-1#178]
Hola buenas es valido definir como el conjunto de todos los subconjuntos finitos de N, como un producto cartesiano? Tipo NxNx....xN, básicamente el producto cartesiano de N, n veces, con n un natural mayor o igual a 1.

Saludos

**Comment by Dvckhv** [comment]
Hola! Lo que mencionas corresponde al **conjunto potencia** visto en clases, en particular en la clase de teoria de conjuntos

**Comment by tremendapithon** [comment]
Pero estoy usando un producto cartesiano de AxA, incluso he visto algunas demostraciones que lo hacen así. Entonces, a que se refiere con lo que preguntan en la A?


Saludos

**Comment by tremendapithon** [comment]
![Screenshot_20240530_093539_Edge.jpg](https://github.com/IIC1253/IIC1253-2024-1/assets/119064623/18d2f005-47bb-46ed-8fb7-a098439cf9c5)

eso me sale en Internet

**Comment by tremendapithon** [comment]
El conjunto potencia de P(N) serian todos los subconjuntos infinitos de N, y a su vez este conjunto es no numerable. Al menos pense este conjunto para la b

**Comment by Dvckhv** [comment]
Por como lo planteaste, entendí que te referias al conjunto potencia, este en particular incluye a todos los subconjuntos, no a los infinitos en particular.

**Comment by tremendapithon** [comment]
Entonces es valido usar ese conjunto y demostrar el inciso.

**Comment by Dvckhv** [comment]
Me referia a que este conjunto potencia no incluye solo a los subconjuntos infinitos sino que a todos, los productos cartesianos son un subconjunto del conjunto potencia y los conjuntos infinitos también.

**Comment by tremendapithon** [comment]
claro, pero es valido hacer NxNxNx....xN para la 2a, y en base a ese conjunto buscar una inyección entre ese conjunto y los naturales.

**Comment by Dvckhv** [comment]
Claro, pero de esa forma te faltarian todos lossubconjuntos finitos de tamaño n-1
